### PR TITLE
feat(analytics-js-integrations): add Wingify device mode integration

### PR DESCRIPTION
**Pull request by zeeshan-vwo**
## Summary

Introduce Wingify as a new device mode destination with the V3.0 async smartcode loader and full SDK registration across integrations, plugins, and legacy mappings.

This integration mirrors the existing VWO destination pattern, rebranded for Wingify:

- New `Wingify` integration under `analytics-js-integrations` with `track`, `identify`, and experiment-viewed support
- V3.0 smartcode loader loading from `https://edge.wingify.net/tag/{accountId}.js`
- Wingify-specific globals: `window.WINGIFY`, `window._wingify_code`, `_wingify_exp`
- Registration in `integrations/index.js`, `Destinations.ts`, CDN file name map, v1.1 cname/client mappings, and legacy config-to-integration names
- Unit tests for browser and utils

## Motivation and context

Wingify is being introduced as a standalone destination alongside VWO. Customers configuring Wingify in RudderStack need a dedicated integration that loads the Wingify smartcode and forwards RudderStack events/traits to the Wingify SDK.

## Test plan

- [x] Wingify unit tests pass (`browser.test.js`, `utils.test.js`)
- [x] `analytics-js-integrations` test suite passes (986 tests)
- [x] `analytics-js-plugins` test suite passes (470 tests)
- [x] Lint passes on affected packages (0 errors)
- [x] Wingify CDN bundles build successfully (legacy + modern)

## Linear task (optional)

Linear task link

## Cross Browser Tests

Please confirm you have tested for the following browsers:

- [x] Chrome
- [x] Firefox
- [x] IE11

## Sanity Suite

- [x] All sanity suite test cases pass locally

## Security

- [x] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

The integration only loads scripts from the configured Wingify CDN endpoint (`edge.wingify.net`) using the customer-provided account ID, consistent with other device mode destination loaders in this repo.

## Summary by CodeRabbit

* **New Features**
  * Added Wingify integration with event tracking, user identification, experiment capture, and optional native SDK loading.
  * Supports sanitized event and attribute names and revenue tracking for completed orders.

* **Tests**
  * Added coverage for initialization, tracking, identification, sanitization, SDK loading, and validation edge cases.

* **Chores**
  * Registered Wingify across integration and display-name configurations.

### CODE REVIEW

**Comment by zeeshan-vwo** [comment]
Hi @shekhar-rudder, please review this PR. Let me know any other details are needed.

**Comment by SKannaniOS** [comment]
@dependabot rebase

File: packages/analytics-js-integrations/src/integrations/Wingify/browser.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    const wingifyIntgConfig = getDestinationOptions(this.analytics.loadOnlyIntegrations);
+    if (wingifyIntgConfig?.loadIntegration) {
+      const account_id = this.accountId;
+      const settings_tolerance = this.settingsTolerance;
+      loadNativeSdk(account_id, settings_tolerance);
+    } else {
+      logger.info('loadIntegration flag is disabled');
+    }
+
+    window.WINGIFY = window.WINGIFY || [];
```
Comment (dciccale) [inline packages/analytics-js-integrations/src/integrations/Wingify/browser.js:44]: Could we update this integration to use the documented `window.Wingify` global? Current Wingify examples initialize `window.Wingify`, so calls queued on `window.WINGIFY` will not reach SmartCode unless Wingify confirms an alias. Please add a contract test against `window.Wingify` instead of an uppercase local shim.

File: packages/analytics-js-integrations/src/integrations/Wingify/browser.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    window.WINGIFY.push([
+      'onVariationApplied',
+      data => {
+        if (!data) {
+          return;
+        }
+        const expId = data[1];
+        const variationId = data[2];
+        const experiment = window._wingify_exp?.[expId];
```
Comment (dciccale) [inline packages/analytics-js-integrations/src/integrations/Wingify/browser.js:80]: The [Wingify v3 documentation](https://developers.wingify.com/v3/reference/variation-applied) reads campaign metadata from `window.Wingify.campaigns`. I also verified current SmartCode bundles from VWO and Wingify expose `window.Wingify.campaigns` and `_vwo_exp`, but not `_wingify_exp`. This lookup therefore returns `undefined`, and the callback exits without sending `Experiment Viewed`. Could we use the documented callback fields or `window.Wingify.campaigns` and add an experiment callback test?

File: packages/analytics-js-integrations/src/integrations/Wingify/nativeSdkLoader.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  }
+
+  let finished = false;
+  const o = n.currentScript;
+  let config = { sT: settings_tolerance, hES: hide_element_style, hE: hide_element };
+
+  try {
+    const storedConfig = localStorage.getItem(`_wingify_${account_id}_config`);
+    if (storedConfig) {
+      config = Object.assign(JSON.parse(storedConfig), config);
```
Comment (dciccale) [inline packages/analytics-js-integrations/src/integrations/Wingify/nativeSdkLoader.js:27]: The official [Async v3.0 SmartCode example](https://developers.wingify.com/v2/docs/fme-unified-experimentation-identity-synchronization) lets the stored configuration override the default values. This code uses the reverse order. Was this change intentional for the RudderStack integration?
